sos/archive.py

```diff
@@ -342,12 +342,14 @@ def add_file(self, src, dest=None):
                 # path case
                 try:
                     shutil.copy(src, dest)
-                except IOError as e:
+                except (IOError, OSError) as e:
```

Comment:
I would rather see separate "except OSError" for two reasons:

- with the current change, a fail to copy `/sys/*` or `/proc/*` wont be logged
-for other paths and `OSError`, we get two info logs, both the `caught` and `File not collected`

Reply:
Agreed.

Reply:
IOError and OSError separated